mapeditor/Animation.cpp

```diff
@@ -686,6 +689,44 @@ std::shared_ptr<QImage> Animation::getImage(size_t frame, size_t group, bool ver
 	return nullptr;
 }
 
+void Animation::exportBitmaps(const bfs::path& path, bool prependResourceName) const
+{
+	if (images.empty())
+	{
+		logGlobal->error("Nothing to export, animation is empty");
+		return;
+	}
+
+	boost::filesystem::path actualPath = path / "Sprites" / name;
+	boost::filesystem::create_directories(actualPath);
```

Comment:
why not to use QT stuff instead of boost?

Reply:
mainly historical reasons... but since we use both libraries can I keep the boost version?

Reply:
I propose to leave the current implementation with boost. I would rather skip this step and learn directly the quirks of std::filesystem::path that we will get with C++17 than the QT's.